js/index.js

```diff
@@ -0,0 +1,29 @@
+const menu = document.querySelector('.menu');
+const logo = document.querySelector('header h3');
+const hamburger = document.querySelector('#openmenu');
+const closeHamburger = document.querySelector('#closemenu');
+const ul = document.querySelector('.nav-links');
+const navLinks = document.querySelectorAll('.nav-links li');
+const menuBg = document.querySelector('.menu-bg');
+const lis = Array.from(navLinks);
+
+function toggler() {
+  logo.classList.toggle('hide');
+  ul.classList.toggle('hide');
+  menuBg.classList.toggle('remove');
+  document.body.classList.toggle('no-scroll');
+}
+
+menu.addEventListener('click', () => {
+  toggler();
+  hamburger.classList.toggle('remove');
+  closeHamburger.classList.toggle('remove');
+});
+
+for (let i = 0; i < lis.length; i += 1) {
+  lis[i].addEventListener('click', () => {
+    toggler();
+    closeHamburger.classList.add('remove');
+    hamburger.classList.remove('remove');
+  });
+}
```

Comment:
### To match the Figma design:
- [ ] Kindly change the `font-weight` to be `500` for the menu links instead of `bold`.
- [ ] Kindly change the color of the close button to `#67798E`.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/167503881-eb918a50-9a5c-44f0-b100-9a9e680cc3c7.png) | ![image](https://user-images.githubusercontent.com/24703171/167503935-49fc29a7-5385-4673-848d-a1fdd01aaa05.png) |